tests/hubapi/demohub/plugins/uniprot/dumper.py

```diff
@@ -9,7 +9,7 @@
 
 class UniprotSpeciesDumper(FTPDumper):
 
-    SRC_NAME = "uniprot_species"
+    SRC_NAME = "uniprot"
```

Comment:
any reason for this change here? Same for the `UniprotSpeciesUploader` below as well. @remoteeng00

Reply:
Just because the plugin name is `uniprot` so  changed to match this name, I'll revert if you think it should not be changed.

Reply:
I thought it should share the same name with data plugin name, but it's ok if keeping old name, this change is reverted.